### DISCUSSION TITLE
【求助】谁能告诉我鸟为什么会飞？

### DISCUSSION THREAD

**Discussion by phaeton66** [share121/inter-knot#1228] [其他]
最近天天梦见一个白发男子问我鸟为什么会飞，还说什么终焉的陨星落下什么什么的，乱七八糟，真的烦透了
![70d92690fcaf2aeadb3bac66dfbe5b5c_4904898721005491436](https://github.com/user-attachments/assets/8ec78b1a-88f9-4df9-9889-af678ad45d9c)

**Comment by XiaoFffeng** [comment]
不会飞就会·被我变成烧烤（doge）

**Comment by Inter-Knot-User** [comment]
因为那位男子拿着天火在后面追

**Comment by GanlinSama** [comment]
啊，这题我会

**Comment by GanlinSama** [comment]
因为他们「必须」飞上天际。

当终焉的陨星在白垩纪落下，唯有自由的鸟儿才能逃出既定的灭亡。

**Comment by Larksonguesinaspic** [comment]
Parce qu’ils doivent voler dans le ciel.

**Comment by gaojiaobei** [comment]
其实你有卡斯兰娜血脉，那白发男子其实是你祖宗，人是带你去·拯救世界的

**Comment by Kuuga2000skills** [comment]
> 最近天天梦见一个白发男子问我鸟为什么会飞，还说什么终焉的陨星落下什么什么的，乱七八糟，真的烦透了 ![70d92690fcaf2aeadb3bac66dfbe5b5c_4904898721005491436](https://private-user-images.githubusercontent.com/178054209/356844095-8ec78b1a-88f9-4df9-9889-af678ad45d9c.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjMzNDcyNDIsIm5iZiI6MTcyMzM0Njk0MiwicGF0aCI6Ii8xNzgwNTQyMDkvMzU2ODQ0MDk1LThlYzc4YjFhLTg4ZjktNGRmOS05ODg5LWFmNjc4YWQ0NWQ5Yy5qcGc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwODExJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDgxMVQwMzI5MDJaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT05ODBhNmM1ZWYxNDc0NDFiZTliMWZmOTRhNWZjMTcwYmNiMGY1Mzk4ZmQ1ZDY5MTIzNmIyZjZkZWFmMGJiMjgyJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.4aVqv8jksnNtkL-awm9sQMAVTNDgCKPRGgSgcUKK4JA)
不会飞的小鸟身来就是要被吃掉的

**Comment by YukariZZZ** [comment]
看到后面那长发女子了吗，她叫苏是这位白发男子的老婆，你可以找她寻求帮助。

**Comment by Dream-Dyer** [comment]
嗯呢嗯呢

**Comment by qbsozo** [comment]
反問他島為什麼會炸

**Reply by phaeton66** [reply]
渡鸦小姐向您发来了决斗请求

**Comment by su2569** [comment]
6

**Comment by EhuanyingK** [comment]
因为它们「想要」飞上天际

**Comment by 123-Invincibleswordsman** [comment]
鸟儿要么拥抱天空，要么沉浸油锅

**Comment by dogofvita** [comment]
因为不会再有第二个爱莉希雅了（）

**Comment by AKT505** [comment]
请问是前文明的凯文先生吗

**Reply by phaeton66** [reply]
我糙，盒，凯文错了凯文不该在网上口嗨的

**Comment by keerdun** [comment]
因为总有一天会从梦中醒来（雾

**Comment by SeeleVollereid** [comment]
鸟为什么会飞？
这里可以指出鸟适于飞行的形态结构特点：

1.体型，呈流线型

2.体表覆羽，正羽用于飞行，绒羽用于保温

3.前肢变成翼

4.胸肌发达

5.骨骼：具有龙骨突，骨轻、薄、坚固，长骨中通

……开玩笑的。只有当终焉的陨星在白垩纪降下，唯有自由的鸟儿才能跳出既定的灭亡

**Reply by SeeleVollereid** [reply]
来补个档

**Comment by SeeleVollereid** [comment]
鸟为什么会飞?
因为它们「必须」飞上天际。
当终焉的陨星在白垩纪降下，唯有自由的鸟儿才能跳出既定的灭亡。
有些人的飞翔，正是为了坠落。
大家都认为，伊卡洛斯是出于自大，在一场意外中不幸丧生。
但如果这在很多年以前，就已经是他想要去做的事呢？
在他很小的时候，就亲口这样说过——「我将飞向上天际，并且以坠落迎接自己的胜利」。
他想证明的只有一件事——
「我飞到了太阳的面前——没有任何人达到过的地方。」
「所以，或许也有人能够将我跨越。」
那么，鸟为什么会飞？
因为它们曾经见到过，最初的鸟以一颗高贵如月的心脏，试图触摸天顶，却坠亡在了地面上。
因为它们曾经见到过，后来的鸟同样做出类似的尝试，并且越飞越高——
所以此刻，鸟才依然盘旋于天际。
自己是哪一只鸟呢？
男人并不知道。但不知为何，他在此时偏偏想起了一颗滚落于地面的篮球。
伊卡洛斯并没有失败。他的坠落是飞行的成果，是另一种胜利的终点。
纵然这是一种格局极其狭窄的看法，是属于浪漫主义者的一厢情愿……
可这或许也正是唯一一种能使世界得以运转的规则。
「没错，为了不让太阳落下，我飞上天际，将你们的光芒夺去。我将因之融化，坠落于海面。但要想将其夺回，你，你们——必须飞到比我更高的地方。」【补档+1】

**Comment by Phaethon-0019** [comment]
你告诉他，曾经，当第一只鸟儿飞上天际，那是世界对他的期许，然自此以后，不再有任何雏鸟坠亡大地……

**Comment by demolute** [comment]
为了坠只因

**Comment by I-m-Miss-Lucy-s-dog** [comment]
是這樣的 問問題的人只需要問 而我們回答的人要考慮的就很多
